packages/launchpad/src/launchpad.tsx

```diff
@@ -81,11 +81,13 @@ const Launchpad = ( {
 				<ShareSiteModal setModalIsOpen={ setShareSiteModalIsOpen } site={ site } />
 			) }
 			<LaunchpadInternal
+				site={ site }
 				siteSlug={ siteSlug }
 				checklistSlug={ checklistSlug }
 				taskFilter={ taskFilter }
 				useLaunchpadOptions={ launchpadOptions }
 				launchpadContext={ launchpadContext }
+				enableAutoTracking
```

Comment:
Non-blocking thought: this may be more descriptive if we call it `enableInternalTracking`. It's still somewhat vague, but I think we could capture what it's enabling a bit more.

Reply:
I agree with the suggestion but it is a temporary flag that I introduced just to reduce the PR size, I am removing this flag on the PR related to that I am moving the onboarding tasks tracking https://github.com/Automattic/wp-calypso/pull/87301, commit https://github.com/Automattic/wp-calypso/pull/87301/commits/331dc248c6ecb2966d2603457c0bb6abf07019a5